crypto/curve25519/curve25519.c

```diff
@@ -777,10 +778,9 @@ static void table_select(ge_precomp *t, const int pos, const signed char b) {
 //
 // Preconditions:
 //   a[31] <= 127
-void x25519_ge_scalarmult_base(ge_p3 *h, const uint8_t a[32]) {
+void x25519_ge_scalarmult_base(ge_p3 *h, const uint8_t a[32], int use_adx) {
```

Comment:
Our goal is to move all runtime processor feature detection to Rust. We've already removed all the C functions for runtime detection.